src/modules/cmdpal/Microsoft.CmdPal.UI/Settings/SettingsWindow.xaml.cs

```diff
@@ -35,10 +39,33 @@ public SettingsWindow()
         WeakReferenceMessenger.Default.Register<QuitMessage>(this);
     }
 
+    // Handles NavigationView loaded event.
+    // Sets up initial navigation and accessibility notifications.
     private void NavView_Loaded(object sender, RoutedEventArgs e)
     {
+        // Delay necessary to ensure NavigationView visual state can match navigation
+        Task.Delay(500).ContinueWith(_ => this.NavigationViewLoaded?.Invoke(), TaskScheduler.FromCurrentSynchronizationContext());
+
         NavView.SelectedItem = NavView.MenuItems[0];
         Navigate("General");
+
+        if (sender is NavigationView navigationView)
+        {
+            // Register for pane open/close changes to announce to screen readers
+            navigationView.RegisterPropertyChangedCallback(NavigationView.IsPaneOpenProperty, AnnounceNavigationPaneStateChanged);
+        }
+    }
+
+    // Announces navigation pane open/close state to screen readers for accessibility.
+    private void AnnounceNavigationPaneStateChanged(DependencyObject sender, DependencyProperty dp)
+    {
+        if (sender is NavigationView navigationView)
+        {
+            UIHelper.AnnounceActionForAccessibility(
+            ue: (UIElement)sender,
+            (sender as NavigationView)?.IsPaneOpen == true ? "Navigation pane opened" : "Navigation pane closed",
```

Comment:
Should "Navigation page opened" and "Navigation pane closed" be Resources so we can localize them?